CHANGELOG.md

```diff
@@ -9,6 +9,17 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Breaking
 
+- When using the cluster module, you must now call
+  `promclient.setupClusterWorker()` from each cluster worker.
+
+  Long explanation: v13.2.0 introduced a change from v13.1.0 that broke cluster
```

Comment:
sounds like we should revert that change, release a patch, then a new major with that and this?

Reply:
That's a good idea, I can release a v13.2.1 with it reverted.

Reply:
It looks like v13.2.1 was never released so would it be possible to get this MR merged? The cleaner API is nice.